rtcclient/client.py

```diff
@@ -77,56 +79,24 @@ def get_rtc_obj(self):
         return self
 
     def _get_headers(self):
-        if self.jazz is True:
-            _allow_redirects = True
-        else:
-            _allow_redirects = False
-
+        self.http = httplib2.Http()
```

Comment:
Why introducing this new lib?

We've encapsulated several HTTP request methods to support logging and special short circuits. Better to keep them consistent.